.dockerignore

```
@@ -0,0 +1,8 @@
+.circleci/
+tests/
+venv/
+data/
+tests/
+dumps/
+pyenv/
+__pycache__/
```

Ignore all `__pycache__` dirs below root
```suggestion
*/__pycache__ 
```